### DISCUSSION TITLE
Pre-release channel is now live!

### DISCUSSION THREAD

**Discussion by gcampbell-msft** [microsoft/vscode-makefile-tools#469] [Announcements]
Hi everyone!

I'm happy to announce that we now support a pre-release channel for vscode-makefile-tools! This pre-release channel will be kept up to date with our most recent, hot-off-the-press changes for you all to try out and give feedback on. 

We hope that this is as exciting for you all as it is for us. Please try out the pre-release channel by navigating to the Makefile Tools extension in VS Code and pressing "Switch to Pre-Release Version". 

Thanks!

**Comment by GerryOnGithub** [comment]
C# v2.39.28 (pre-release)
C# Dev Kit v1.9.8 (pre-release)
IntelliCode for C# Dev Kit v2.1.11

:-( 
No such luck

**Reply by gcampbell-msft** [reply]
@GerryOnGithub Are you hitting an issue? I'm not understanding your comment, please let me know if you have any issues with the Makefile Tools Extension. Thanks.
